src/DotLiquid/Tags/Include.cs

```diff
@@ -60,9 +60,9 @@ public override void Render(Context context, TextWriter result)
                 foreach (var keyValue in _attributes)
                     context[keyValue.Key] = context[keyValue.Value];
 
-                if (variable is IEnumerable)
+                if (variable is IEnumerable enumerable && !(variable is string))

```

Comment:
I see you added new tests for include tag, but wouldn't this code line change the behavior compared to before? Should this also be behind the 2.4 syntax compatibility check?

Reply:
I agreed with you on the for loop that people might have expected to be able to iterate through a string. but to include a template for a single char is not even a thought.